src/components/RequestPanel/RequestStatus.js

```diff
@@ -182,34 +168,34 @@ function SingleTx({ status }) {
   )
 }
 
-function SigningStatusInfo({
+function RequestStatusInfo({
```

Comment:
I find these changes afterwards to be more confusing—most of the time, we will be sending transactions so using solely the "request" language is odd.

Perhaps we can either:
- Use the `isTx` flag to help us build messages
- Generalize the language so we don't have to choose one or the other (e.g. "Everything was successful" vs. "Transaction/Request was successful")